code/game/objects/items/weapons/cigs_lighters.dm

```diff
@@ -143,10 +143,10 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	var/lighting_text = null
 	if(istype(O, /obj/item/weapon/weldingtool))
 		lighting_text = "<span class='notice'>[user] casually lights the [name] with [O], what a badass.</span>"
+	else if(istype(O, /obj/item/weapon/lighter/greyscale)) // we have to check for this first -- zippo lighters are default
+		lighting_text = "<span class='notice'>After some fiddling, [user] manages to light their [name] with [O].</span>"
 	else if(istype(O, /obj/item/weapon/lighter))

```

Comment:
You should just correct the path here and then it just works. It's supposed to be lighter/zippo.